internal/config/config.go

```diff
@@ -0,0 +1,96 @@
+package config
+
+import (
+	"fmt"
+
+	clierr "github.com/10gen/realm-cli/internal/cli/err"
```

Comment:
to my earlier comment about this package - having to prefix your imports is typically antipattern in go and would like to avoid it if possible in the cli.

Reply:
thoughts on a good name for a shared errors package then?  `commons.Error`?  I don't want to re-use `errors` with it in conflict with go's package.  open to ideas here, but I explained above the desire to keep it out of the `cli` package (so that `ui` can leverage it)

Reply:
`clierr` is no more :)